app/views/users/show.html.erb

```diff
@@ -1,5 +1,49 @@
 <section class="user-section">
-  <h2><%= "Name: #{@user.name}" %></h2>
+  <%= flash[:message] %>
+
+<%= "#{@show_user.name.upcase }'s PROFILE'" %>
+
+<% if current_user == @show_user %>
+
+  <h1>Send A Friend Request</h1>
+
+  <%= form_for :friendship, url: friendships_path do |f|%>
+    <%= f.select :confirmer_id, 
+      User.all.collect { |u| [ u.name, u.id ] if u.id != current_user.id }.select(&:present?), 
+      {:prompt => 'Select Person'}, class: "select_friend"
+    %>
+
+    <%= f.submit "Send Friend Request" %>
+  <% end %>
+
+
+  <h1>Friend Requests From Others</h1>
+  <ul>
```

Comment:
I don't suggest you keep the friendship invitations here. It will be difficult for someone to find them. Better create a new menu for that (maybe called Notifications?)